src/SlnGen.Build.Tasks.UnitTests/SlnProjectTests.cs

```diff
@@ -182,8 +182,16 @@ private sealed class TestProject : IDisposable
         {
             private const string TemplateProjectPath = @"TestFiles\SampleProject.csproj";
 
+            private readonly Dictionary<string, string> _savedEnvironmentVariables = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+
             private TestProject(string fullPath, IDictionary<string, string> globalProperties)
             {
+                SetEnvironmentVariables(new Dictionary<string, string>
+                {
+                    { "Configuration", null },
+                    { "Platform", null },
+                });
```

Comment:
This can't be controlled by AppVeyor configurations ?

Reply:
Yeah they are set as part of our build.  We need them to be set for other things but not set when these tests run...